### DISCUSSION TITLE
使用vercel作为服务器部署，今天中午，网页中文章全部消失，notion的文章无法显示

### DISCUSSION THREAD

**Discussion by psycounselor** [tangly1024/NotionNext#1979] [Q&A]
**描述bug**
使用vercel作为服务器部署，今天中午，网页中文章全部消失，notion的文章无法显示

**复现步骤**
[出现这个bug的操作步骤](https://www.heals.top/)

**期望的正常结果**
希望可以恢复正常

**截图**

https://www.heals.top/
**环境**

- 操作系统: [windows]
- 浏览器 [edge]
- NotionNext版本 [最新版]

**Comment by psycounselor** [comment]
![日志](https://heals-top.oss-cn-hangzhou.aliyuncs.com/oss://heals-top%E5%B1%8F%E5%B9%95%E6%88%AA%E5%9B%BE%202023-08-23%20170120.png)

**Comment by dadaoerv** [comment]
盲猜，page id变了。只要你notion笔记内容没问题。就不会丢。如果是notion笔记都丢了，也不关本项目的事情。

**Comment by psycounselor** [comment]
数据是没丢，关键是page id也没变，重新部署，主页也是不显示内容

**Comment by happyzhangyyds** [comment]
> 数据是没丢，关键是page id也没变，重新部署，主页也是不显示内容

刚刚访问您的主页，似乎问题已经得到解决。

**Comment by psycounselor** [comment]
问题已自行处理

**Comment by Flyingpen** [comment]
> 问题已自行处理

请问你是如何处理的？我也遇到了相同的问题，文章本来是170多，现在只有90多篇，今天更新的文章，也死活刷新不出来，时间不仅停留在9月底，以前的一些文章也刷不出来了。

**Comment by psycounselor** [comment]
@Flyingpen 咱们的问题应该不是同一个，我的处理方法是，notion页面重新赋值一份生成新的id，再绑定vercel，部署后，即可恢复正常。国庆期间，vercel被墙，刚更换了域名，抢救回来。新地址：www.hears.top
